debug.log

```diff
@@ -0,0 +1,2 @@
+[1205/234319.653:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
Can you please remove this file coz it's not relevant to the issue.

Reply:
Done